.github/workflows/ci_cron_weekly.yml

```diff
@@ -134,8 +134,8 @@ jobs:
                                   python3-ply \
                                   python3-venv \
                                   cython3 \
-                                  libwcs7/bullseye-backports \
-                                  wcslib-dev/bullseye-backports \
+                                  libwcs7/bookworm-backports \
+                                  wcslib-dev/bookworm-backports \
```

Comment:
No bookworm-backports yet so you can remove the `/bookworm-backports` here and the other line above.

Reply:
Can't just leave it in to future-proof?